### DISCUSSION TITLE
Accordion expand on hover

### DISCUSSION THREAD

**Discussion by tianhuil** [mantinedev/mantine#1586] [Feature requests]
@rtivital 

Is there a way for Accordion to expand on hover?  Ideally, it would take a `trigger='hover'` or `trigger='click'` (default) argument [like with Menu](https://mantine.dev/core/menu/#show-menu-on-hover).

**Comment by rtivital** [comment]
You can control accordion state on your side – https://mantine.dev/core/accordion/#control-state
We will not add hover feature as a part of the library

**Reply by tianhuil** [reply]
Thanks for the workaround.
